### PR TITLE
TM-4989 Updates to EAS export and search UI

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
[Ticket](https://metaphase.atlassian.net/browse/TM-4989)

- [x] Implement Loading Toast
- [x] Implement Error Toast
- [x] Update export button text
- [x] Update UI so the search buttons for text look more like buttons
- [x] Ensure all Toast and Alert Notifications are working 

image

### CODE REVIEW

File: src/sass/_employeeSearch.scss [hunk 1]
```diff
@@ -63,13 +63,42 @@
           height: 28px;
           max-width: none;
           width: 240px;
+          padding: 0 35px 0 10px;
         }
         .searchbar {
           .text-input-wrapper {
-            > span {
+            // option 1
+/*            span {
```
Comment (elizabeth-jimenez) [inline src/sass/_employeeSearch.scss:71]: two style options for the search buttons

**Comment by scott062** [comment]
`height: 26px` fixed it for me

**Comment by scott062** [comment]
I like your option 1 best (filled in)

**Comment by scott062** [comment]
Abbreviate max. (but can format however you think)

**Comment by scott062** [comment]
Everything else looks good for 🍞 - done with review, just the above comments

**Comment by elizabeth-jimenez** [comment]
> I like your option 1 best (filled in)
> 
> But hover - I'd just do the darker shade of blue to show they're about to select. Cannot remember the other darker button like this but I know we do it somewhere else. The white toggle feels very harsh, thinking more subtle

image
hovering on First Name one

**Review by scott062 — APPROVED** [review]

**Pull request merged**